test/unit/spec/v2/view-builder/utils/CaptchaUtil_spec.js

```diff
@@ -0,0 +1,68 @@
+import $sandbox from 'sandbox';
```

Comment:
sandbox is obsolete pattern. usually we don't need a container or could just use `body`